### ISSUE TITLE
remove ms:distributionLocation

### ISSUE THREAD

**Issue by pennyl67** [pennyl67/LexMeta#8]
Please, remove it; it has been merged with "ms:accessLocation" because it created confusion and programmatically the two made no difference.

**Comment by dlindem** [comment]
I remember the discussion on this. In the LexBib catalogue, at the moment, I use ms:distributionLocation in cases where I am not sure if a URL is a landing page or an access location (these are the two subproperties of ms:distributionLocation, isn't it?)

**Comment by dlindem** [comment]
Yes, we can do that. But I still think it is good to have ms:accessLocation for URL we know for sure it points to a page from where the resource can be downloaded from (so that the machine can display the page or look for the download link on the page), and ms:downloadLocation for URL we know for sure it is a direct download link (so that the machine or the user could directly do "save target as"). In case we are not sure, we could use the neutral superproperty. This is the way I currently use the three properties on LexBib.
